lib/schema.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { schema } = require('@platformatic/service')
+const { version } = require('../package.json')
+
+const rabbitmqHooksSchema = {
+  ...schema.schema,
+  $id: 'rabbitmq-hooks',
+  title: 'Rabbitmq Hooks Config',
+  version,
+  properties: {
+    ...schema.schema.properties,
+    module: { type: 'string' },
+    rabbitmq: {
+      type: 'object',
+      properties: {
+        url: { type: 'string' },
+        exchange: { type: 'string' },
+        routingKey: { type: 'string' },
+        targetUrl: { type: 'string' }
```

Comment:
What method is going to be used? additional headers would also be required.

Reply:
We use `POST` in this PR.